### ISSUE TITLE
oce.plot.ts() time axis improvement: large and small ticks

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1018]
For some time intervals, like the one shown, it can be quite hard to tell which tick corresponds to a label. The simple solution is to have fewer ticks, but I feel a more elegant solution, generally, would be to use small ticks between the labelled ticks. The plot geometry could be otherwise unaltered ... the idea would be to use intermediates ticks that are 2/3 (or something) as long as normal. Any opinions on this, Clark?

<img width="508" alt="screen shot 2016-07-13 at 6 08 31 pm" src="https://cloud.githubusercontent.com/assets/99469/16820069/a07788f4-4925-11e6-843a-3efc7da34a1e.png">

**Comment by dankelley** [comment]
Oh, we have more data now and the auto-scale (below) handles the two-tick-size topic in this case. So the coding is going to be simple.
<img width="511" alt="screen shot 2016-07-14 at 1 09 48 pm" src="https://cloud.githubusercontent.com/assets/99469/16846842/633e2bce-49c4-11e6-8df7-446a2b191481.png">

**Comment by richardsc** [comment]
What you're describing is already implemented for some time ranges, e.g.:
```r
library(oce)
t1 <- Sys.time()
t2 <- t1 + 3 * 86400
t <- seq(t1, t2, 600)
x <- rnorm(t)
oce.plot.ts(t, x)
```
![screen shot 2016-07-14 at 12 12 01](https://cloud.githubusercontent.com/assets/233584/16846887/36d6b590-49bc-11e6-98f4-88357f03d27a.png)

**Comment by richardsc** [comment]
Ha, oops! Nearly simultaneous post! 😄

**Comment by dankelley** [comment]
Fixed in `develop` commit b21e2fd3a832ccde386caf3ee5cd3a81d0d1eda5; a test code (in the test suite) is below
```R
library(oce)
t1 <- Sys.time()
days <- c(0.5, 2.5, 3)
if (!interactive()) png("1018a.png")
par(mfrow=c(length(days), 1))
for (day in days) {
    t2 <- t1 + day * 86400
    t <- seq(t1, t2, 600)
    x <- sin(2*pi*as.numeric(t)/86400)
    oce.plot.ts(t, x, debug=3)
}
if (!interactive()) dev.off()
```

![1018a](https://cloud.githubusercontent.com/assets/99469/17034565/9aebc6fe-4f59-11e6-9bfb-0b6d08552474.png)
